formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -1138,6 +1141,83 @@
+          "housing-act-2004-housing-and-planning-act-2016---empty-dwelling-management-orders",
+          "housing-act-2004-housing-and-planning-act-2016---houses-in-multiple-occupation-licensing",
+          "housing-act-2004-housing-and-planning-act-2016---selective-licensing",
+          "housing-act-2004-housing-and-planning-act-2016---banning-orders",
+          "housing-act-2004-housing-and-planning-act-2016---civil-financial-penalties",
+          "housing-act-2004-housing-and-planning-act-2016---management-orders",
+          "housing-act-2004-housing-and-planning-act-2016---overcrowding-notices",
+          "leasehold-disputes-management---service-charges",
+          "leasehold-disputes-management---administration-charges",
+          "leasehold-disputes-management---appointment-of-manager",
+          "leasehold-disputes-management---right-to-manage",
+          "leasehold-disputes-management---variation-of-leases",
+          "leasehold-disputes-management---breach-of-lease-forfeiture",
+          "leasehold-disputes-management---estate-charges-estate-management-schemes",
+          "leasehold-enfranchisement-extension---flats-lease-extension-",
+          "leasehold-enfranchisement-extension---houses-enfranchisement",
+          "leasehold-enfranchisement-extension---flats-collective-enfranchisement",
+          "leasehold-enfranchisement-extension---right-of-first-refusal",
+          "park-homes---pitch-fee",
+          "park-homes---section-4-any-question-under-act-or-agreement",
+          "park-homes---local-authority-compliance-notice",
+          "park-homes---appeal-against-alteration-of-a-site-licence-condition",
+          "park-homes---local-authority-decision-emergency-action",
+          "park-homes---local-authority-site-licence-refusal-refusal-to-transfer",
+          "park-homes---site-owners-failure-to-deposit-site-rules with la",
+          "park-homes---site-rule-change-consultation-response",
+          "park-homes---park-home-having-a-detrimental-effect",
+          "park-homes---order-relating-to-implied-and-express-terms",
+          "park-homes---recognition-of-residents-association",
+          "park-homes---refusal-orders-against-giving-assigning-home",
+          "park-homes---site-licence-payment-of-annual-fee",
+          "park-homes---order-requiring-written-statement-of-terms",
+          "park-homes---temporary-re-siting-return-of-resited-park-home",
+          "park-homes---entitlement-to-terminate-an-agreement-for-occupation",
+          "rents---market-rent-assured-shorthold-tenancy",
+          "rents---fair-rent",
+          "right-to-buy---rtb-appeals-elderly-persons",
+          "tenant-associations---recognition-of-tenants-association",
+          "tenant-associations---request-for-information"
+        ],
+      },
+      tribunal_decision_decision_date: {
+        type: "string",
+        pattern: "^[1-9][0-9]{3}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[0-1])$",
```

Comment:
This date pattern is hardcoded a few times in this file. Does the jsonnet format let you extract a shared definition?

Reply:
My jsonnet / json-schema knowledge isn't super great, but this feels possible.  I've put this down on my "clean this up afterwards" list for a separate PR.  Thanks!